### ISSUE TITLE
Setting a vision and goals

### ISSUE THREAD

**Issue by Aekez** [Qubes-Community/Contents#5]
Now that it has become official that we are independent from The Qubes OS Project, it also means long-term planning, vision and goals, falls to us to organize. Having a vision for what we hope this place to become, and goals to help archive this vision, is important so that we all work towards the same vision. This also greatly helps new-comers to understand what we are about, by briefly listing our mission, goals, and vision. This is essentially a strategic statement. What our mission and vision statement should look like, and what kind of goals we're setting out to make to transition between these two (now and future), is what this discussion is all about.

Resources if interested in more details, but it is not necessarily needed reads to discuss this. And if so, then even quick read might be sufficient to quickly get a better idea overview of what a strategic statement is about. 
https://en.wikipedia.org/wiki/Strategic_planning
https://en.wikipedia.org/wiki/Mission_statement
https://en.wikipedia.org/wiki/Vision_statement

We shouldn't make this too complicated (it's fine to ignore above links, they are only suggested optional reads), we just need a definition of what we want to become (defined by our vision statement), by reaching targets (our goal statements), through everyday activities (defined by our mission statement). These three should be sufficient to make a decent strategic statement, helping not only ourselves, but also anyone coming here to more quickly grasp what we're all about. 

_**TL:DR**_
**Mission statement** (today) ---> **Goals statement** (transition) ---> **Vision statement** (future)
Defining these three states, helps not only ourselves to better organize, but anyone who quickly want to find out what we're doing here, by being precise and accurate on our strategic statement. Having a clear strategic statement also cuts down on leadership management, and makes it easier for a community to act without leadership. Strategic statements therefore is also a pillar of reason, or backbone, in a community where the goal is to seek a free fluid and organic community.

**Comment by Aekez** [comment]
It's also okay if visions or goals miss and don't succeed. Predicting the future is hard, but nonetheless, the act of trying reduces the risks that are in the dynamic complex environment, as well as internal confusion about what ones collective purpose is all about. What matters here is not to make a perfect prediction of the future, but using this as a driving machine, something to strive towards to. This is in particular important for us, because we try to reduce the need for a single leader, and by being multiple owners, if we have a strategic statement, it is more clear where everyone wants to go about this project.

It's also important to note that those resources have mistakes in them, strategy statements are not only for profit organizations (while I support the wiki idea here, this is definitely an example of a wiki mistake, as strategic statements are important for all kinds and types of organizations).

**Comment by awokd** [comment]
In no particular order and not sure where they fall, but:

- help make it easy for people to contribute to docs
- assist in getting their contributions QAd and submitted (but not get in the way if they want to submit directly to qubes-doc)
- host Qubes related non-official documents/projects or links to them. These are ones that don't fit for inclusion in the official project for some reason.

**Comment by awokd** [comment]
@Aekez is the above enough to write up a basic strategy statement? If so, would you mind?

**Comment by Aekez** [comment]
@awokd 
Sure, and yes it should be enough, I'll try and expand it a bit and also add a bit too, feel free to review it, this is only a draft suggestion, please criticize anything you find wrong about it, or add what you think is missing.

I tried to put in what I think everyone is feeling about the community project, if it's wrong, then it must be corrected to reflect everyone's collective views.

Note that we can always update it to better match in the future, or when things changes. Everyone feel free to keep add and edit, these statements should never be final, if anything changes, or is missing, then so too should the statement be updated.

A question of time-frames. For example, this statement could aim for 1 year out into the future, however may change if anything changes between then and now. Statements typically last 5 years into the future, but we're in early development with a lot of turmoil, so 1 year might be best. So in 1 year, if we're not changing it before then, this will be a time serving as a reminder to review and reflect on the old statement, questioning if it needs change. The time frame can be seen as not a minimum rule, but a maximum rule. The time-frames is purely a reminder to reflect over ones identity, which may have changed, should it be forgotten. So too does the dynamic and complex environment change, and this too can cause changing needs to a statement.

Below is just a sketch, thoughts?

### Statement ###
**Mission** 
As a small fully independent volunteer community, we're only just starting out growing. The focus on everyday tasks is to build and generate new content, creating value, and gathering new community members to help us archive our goals, leading towards our collective vision. Right now we're building a collective community identity, and we hope others would like to join us in order to do that. A lot work right now is being put into shaping the community, as well as deciding what we, as members of the community, can offer in value back to the community, and how we're going to archive it. It is also important that we start doing collective work on Qubes doc drafts, which may be published, and if accepted become official Qubes doc's. Our daily activity is split between help improving the official Qubes doc's, as well as hosting our own content more suitable to be found in the community.    

**Goals** 
- Finding more people interested in creating Qubes content for the independent and unofficial Community, as well as to The Qubes OS Project.
- Ongoing improving work-flows and helping the community to easier help contribute back to the community.
- Improving our quality, we need people who can keep critical eyes and add reviews.
- Staying true to open source where possible, human rights, freedom, and democratic values. 
- While we are unofficial and independent, we also strive to stay true to The Qubes OS Project's vision, this being one of our fundamental pillars.
- Our content must strive to maintain the highest possible security and privacy, as it is possible. It is important that we don't undermine Qubes OS's purpose, but instead help enhance it with more capabilities.
- Providing support and promotion for smaller Qubes projects, which can either anywhere between independent relations to fully integrated into the community. Support may be non-commercial, help finding others to work on the project, or help sharing the projects finished work to the community.

**Vision**
We strive to one day reach a large crowd of Qubes users, allowing optional choices for quality content to improve how everyone uses Qubes OS, made by the community for the community. This may not necessarily be possible within a year from starting out, however this is what we're striving towards to. Having options allowing users to use Qubes OS in ways that better suited and tailored for individual and personal needs, the freedom to use secure software everywhere from unique and rare use-cases, to common and everyday practices. We hope to become a channel, a community, where everyone can gather and work together, for these common goals and vision.

**Comment by awokd** [comment]
Looks good to me, thanks! Want to post it as is? Could maybe try the PR process too, to eat our own dogfood...

**Comment by Aekez** [comment]
@awokd 
great :) I'm a little worried about using such strong words and making promises this big, perhaps we need to lower it a bit, or maybe I'm being too careful with that? It might be a good idea to have everyone look at it, maybe over a couple of days thinking time before we do anything official with it.
It also sounds like a good idea to try some community PR's first ourselves, as you suggest before posting this. We would need to have that worked out first? 

@taradiddles @adubois @one7two99 
What do you guys think about this statement? Does it need updating? fix or changes?

**Comment by taradiddles** [comment]
The statement is nice for a standalone page but it's too long if it's displayed at the top of the qubes-community README.md
- either keep it simple (eg. something like the [landing page](https://github.com/Qubes-Community/Qubes-Community.github.io)). 
- or have the long statement in a separate .md and link to it from both the qubes-community README.md and the landing page.

**Comment by Aekez** [comment]
That's a good idea. So the full version becomes much more like an optional read, but is available if the reader wants to find it.

brief version suggestion:

**About us**
We're a small fully independent volunteer community striving to build or forward end-user content for Qubes OS, such as guides, docs, and scripts. The focus on everyday tasks is to build and generate new content, creating value, and gathering new community members to help us archive our goals, leading towards our collective [vision](https://github.com/Qubes-Community/Qubes-Community/issues/5). Right now we need more people to help us out turning this place from an idea, into a real community hub for end-user Qubes OS content. However we have already started out on a small scale, feel free to take a look around.

**Comment by Aekez** [comment]
For example have a small version like this at the top to introduce, then keep the landing page like it is. If people want to read the full statement, then they can click the vision link (should be kept on a more appropriate page right, but for now the link is just an illustrative example). 

Feel free to modify or add other suggestion versions, we got time to reflect on how it should addressed.

**Comment by Aekez** [comment]
@Qubes-Community/administrators 
I modified it slightly further before adding it to the landing-page to see how it will end up looking. I reverted the first sub-headline with listing 'daily goals' to 'daily tasks' instead, so it isn't being confused with the statement goals. Thoughts of doing it this way?

I also added numbering to headlines, which makes it more easy to distinctive smaller headlines from the bigger ones. Though perhaps numbering headlines are a bit awkward for a landing page? It might work better if we have a logo or something at the top to break the ice a bit. Thoughts of doing it this way?

**Comment by Aekez** [comment]
I'm closing this issue for now, however it may be re-opened if anyone got something to discuss in regards to the Strategic Statement. Any thoughts are welcome, remember this is an open discussion.
